inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -90,8 +93,10 @@ void LayerTestsCommon::Serialize() {
     CommonTestUtils::removeIRFiles(out_xml_path, out_bin_path);
 }
 
-InferenceEngine::Blob::Ptr LayerTestsCommon::GenerateInput(const InferenceEngine::InputInfo &info) const {
-    return FuncTestUtils::createAndFillBlob(info.getTensorDesc());
+InferenceEngine::Blob::Ptr LayerTestsCommon::GenerateInput(const InferenceEngine::InputInfo& info) const {
+    return FuncTestUtils::createAndFillBlob(
+            InferenceEngine::TensorDesc(info.getPrecision(), targetStaticShape,
```

Comment:
i guess better update `GenerateInputs()` method
for dynamic shapes you need to create new input info with defined shapes and pass into `GenerateInput`

Reply:
GenerateInput() is updated by input is empty or not.